### PR TITLE
simple flake8 updates

### PR DESCRIPTION
**Pull request by vphill**
Removed a few imports that didn't seem to be needed and then just updated comment formatting. 

I left alone other suggestions like old style exception raising  and != None vs is not None suggestions.  For another PR.

### CODE REVIEW

**Comment by vphill** [comment]
Ready for review @somexpert @ldko

**Comment by somexpert** [comment]
Are we concerned about having all the first words in the comments capitalized? There are some that aren't.

**Comment by vphill** [comment]
For now I was just trying to push through the "passes flake8" without digging into the actual content.  Might be something we can focus on after a first deployment.  Might be time spent better in other places?

**Comment by somexpert** [comment]
Makes sense. Well in that case, LGTM :thumbsup:

File: aubreylib/resource.py [hunk 1]
```diff
@@ -1,19 +1,10 @@
...
-    # http://docs.python.org/library/json.html
-    import json
-except ImportError:
-    # simplejson 2.0.9 is available for python 2.4+
-    # http://pypi.python.org/pypi/simplejson/2.0.9
-    # simplejson 1.7.3 is available for python 2.3+
-    # http://pypi.python.org/pypi/simplejson/1.7.3
-    import simplejson as json
+import json
+
```
Comment (damonkelley) [inline aubreylib/resource.py:7]: :tada:

**Comment by ldko** [comment]
works for me :+1:

**Pull request merged**